zebra.py

```diff
@@ -0,0 +1,50 @@
+from ophyd import (Component as Cpt, EpicsSignal, EpicsSignalRO, Device)
+
+
+class ZebraPCBase(Device):
+    sel = Cpt(EpicsSignal, 'SEL')
+    start = Cpt(EpicsSignal, 'START')
+    width = Cpt(EpicsSignal, 'WID')
+    step = Cpt(EpicsSignal, 'STEP')
+
+
+class ZebraPCGate(ZebraPCBase):
+    num_gates = Cpt(EpicsSignal, 'NGATE')
+
+
+class ZebraPCPulse(ZebraPCBase):
+    max = Cpt(EpicsSignal, 'MAX')
+    delay = Cpt(EpicsSignal, 'DLY')
+    status = Cpt(EpicsSignalRO, 'INP.STA')
+
+
+class ZebraPositionCompare(Device):
+    arm_status = Cpt(EpicsSignalRO, 'ARM_INP.STA')
+    download_count = Cpt(EpicsSignalRO, 'NUM_DOWN')
+    disarm = Cpt(EpicsSignal, 'DISARM')
+    encoder = Cpt(EpicsSignal, 'ENC')
+    enc_x = Cpt(EpicsSignal, 'ENC1')
+    enc_y = Cpt(EpicsSignal, 'ENC2')
+    enc_z = Cpt(EpicsSignal, 'ENC3')
+    enz_o = Cpt(EpicsSignal, 'ENC4')
```

Comment:
Is it supposed to be `enz_o`? What does it stand for? It seems to be breaking the pattern established for the x/y/x PVs above.

Reply:
this does actually make sense - I can rename this to omega (I guess I just shortened this too much to "o" instead of "omega")

Reply:
My concern was mostly about the `z` letter in `enz_o`. I think it should be `enc_o`.

Reply:
oh! didn't even see that. thanks for catching it.

I still think that maybe o is a bit too terse... will submit a pull request with both changes